keyserver/coniksserver/main.go

```diff
@@ -45,10 +45,10 @@ func main() {
 
 	flag.Parse()
 
-	switch true {
+	switch {
 	case *genConfig:
 		mkConfig(*dir)
-		return
+		fallthrough
```

Comment:
Only because genkey doesn't have a default value, right.  The code still gets executed but on the empty string, so nothing happens.

I guess it's ok with the fallthrough then, but maybe leave a comment saying that we're relying on that. Or not.